components/portal/settings/partials/user-settings.html

```diff
@@ -62,6 +62,20 @@ <h4 class="md-subhead" ng-if="kvEnabled">Reset Zone<br>
           </md-button>
           <small>Requires page refresh to take effect.</small>
         </div>
+        <h4 class="md-subhead">Reset In-Browser State<br>
+          <small>Clear in-browser session storage and local storage. This is perfectly safe and may cause some changes to take effect faster.</small>
+        </h4>
+        <div layout="column" layout-gt-xs="row">
+          <md-button class="md-primary md-raised" ng-click="resetAll()">
+            Reset All
+          </md-button>
+          <md-button class="md-accent md-raised" ng-click="resetSession(); reload();">
+            Reset Session
+          </md-button>
+          <md-button class="md-warn md-raised" ng-click="resetLocal(); reload();">
+            Reset Local
```

Comment:
Only just now remembering this: I think there had been some discussion of simplifying this for users, offering only the "Reset all" behavior?